README.md

```diff
@@ -1,70 +1,137 @@
-# Cluster Version Operator (CVO)
-
-## Building and Publishing CVO
-
-```sh
-./hack/build-image.sh && REPO=<your personal repo (quay.io/ahbinavdahiya | docker.io/abhinavdahiya)> ./hack/push-image.sh
+# Cluster Version Operator
+
+The Cluster Version Operator (CVO) is one of the [Cluster Operators](https://github.com/openshift/enhancements/blob/master/dev-guide/operators.md#what-is-an-openshift-clusteroperator)
+that run in every OpenShift cluster. CVO consumes an artifact called a _"release
+payload image,"_ which represents a given version of OpenShift and contains the
+manifests needed for all Cluster Operators to function. CVO reconciles
```

Comment:
nit: we're introducing "manifests" here, and it might be worth adding a bit more context like "resource manifests" or a parenthetical explaining that manifests are requesting particular configuration for particular Kubernetes resources or something like that.